src/make_table_example.F90

```diff
@@ -310,12 +326,10 @@ program make_table_example
                          eos_variables(rhoindex),eos_variables(tempindex),eos_variables(yeindex),ns,ng
                     stop
                  endif
-                 if (.not. do_transport_opacities) then
-                    if (local_delta(ns,ng).ne.local_delta(ns,ng)) then
-                        write(*,"(a,1P3E18.9,i6,i6)") "We have a NaN in scat delta", &
-                            eos_variables(rhoindex),eos_variables(tempindex),eos_variables(yeindex),ns,ng
-                        stop
-                    endif
+                 if (local_delta(ns,ng).ne.local_delta(ns,ng)) then
+                    write(*,"(a,1P3E18.9,i6,i6)") "We have a NaN in scat delta", &
```

Comment:
There may have been a conflict here at one point.  Please incorporate the `do_transport_opacities` back into/into your version